auction9-frontend/src/app/app-routing.module.ts

```diff
@@ -23,6 +24,10 @@ const routes: Routes = [
     path: 'wonAuctions',
     component: WonAuctionsComponent
   },
+  {
+    path: 'bid/:id',
```

Comment:
If I understood correctly, we use auctions and bid for the same thing, so let's choose and use only one of them, synonyms will make a confusion.

Reply:
Correct, we will use 'auction' instead of 'bid'.
I'll fix it now